src/table/cells/handle-key-press.js

```diff
@@ -54,17 +54,47 @@ export const preventDefaultBehavior = (evt) => {
   evt.preventDefault();
 };
 
-const handleKeyPress = (evt, rootElement, rowIndex, colIndex, cell, selState, selDispatch) => {
+export const moveFocus = (evt, rootElement, rowIndex, colIndex, selState) => {
```

Comment:
Yeah I'd say this should have seperate tests, basically the same as you have now for `handleKeyPress`. Only on tests is enough, and that gets all the smaller functions as well: `preventDefaultBehavior`, `removeCurrentFocus`, `moveToNextFocus`

Reply:
sure